plugins/python/_hexchat.py

```diff
@@ -97,21 +102,24 @@ def get_info(name):
         # Surely there is a less dumb way?
         ptr = repr(ret).rsplit(' ', 1)[1][:-1]
         return ptr
+
     return __decode(ffi.string(ret))
 
 
 def get_prefs(name):
     string_out = ffi.new('char**')
     int_out = ffi.new('int*')
-    type = lib.hexchat_get_prefs(lib.ph, name.encode(), string_out, int_out)
-    if type is 0:
+    _type = lib.hexchat_get_prefs(lib.ph, name.encode(), string_out, int_out)
+    if _type == 0:
         return None
-    elif type is 1:
+
+    if _type == 1:
         return __decode(ffi.string(string_out[0]))
-    elif type is 2 or type is 3:  # XXX: 3 should be a bool, but keeps API
+
+    if _type in (2, 3):  # XXX: 3 should be a bool, but keeps API
         return int_out[0]
-    else:
-        assert False
+
+    raise ValueError('Unknown storage type for pluginpref')
```

Comment:
Revert this. If any other value exists its a programmer error on hexchat's side, thus an assertion is appropriate.

Reply:
In which case a message with the assert should be warranted. What about `raise AssertionError("Unknown type stored for pluginpref")`? that way its more than just an assertion error with a stack trace

Reply:
Thats fine.

Reply:
Switched to `raise AssertionError`